docs/github.md

```diff
@@ -1,34 +0,0 @@
-# GitHub Actions params fetching
```

Comment:
This is another forgotten cleanup. This file is now out of date after removal of v1